pkg/arvo/sys/lull.hoon

```diff
@@ -904,6 +904,12 @@
   ::    payload: semantic message contents
   ::
   +$  plea  [vane=@tas =path payload=*]
+  ::
+  +$  message
+    $%  [%plea plea]
+        [%boon payload=*]
+        [%naxplanation =message-num =error]
```

Comment:
Tempted to say this should be `%naxp` to keep it a direct atom.

Reply:
`%naxplanation` is the same as in `$ack`, changing that also will be annoying.